sleap/io/dataset.py

```diff
@@ -689,14 +691,14 @@ def __getitem__(self, key, *args) -> Union[LabeledFrame, List[LabeledFrame]]:
         else:
             raise KeyError("Invalid label indexing arguments.")
 
-    def get(self, *args) -> Union[LabeledFrame, List[LabeledFrame]]:
+    def get(self, *args, **kwargs) -> Union[LabeledFrame, List[LabeledFrame]]:
```

Comment:
See comments above re: `__getitem__`.

Moving most of the contents of `__getitem__` to this `get()` method would also be a good opportunity to properly typehint here since `*args, **kwargs` make it really difficult to know how to call methods and should be avoided where possible.